backend/entities/Claim.cs

```diff
@@ -21,6 +21,11 @@ public class Claim : BaseEntity
         /// <value></value>
         public string Name { get; set; }
 
+        /// <summary>
+        /// get/set - The GUID that identifies this Role in Keycloak.
+        /// </summary>
+        public Guid? KeycloakRoleId { get; set; }
```

Comment:
Using a subordinate property to link with keycloak to free us from some of the issues that integration with keycloak was causing.